dcc-storage-server/src/main/java/org/icgc/dcc/storage/server/controller/ObjectUploadController.java

```diff
@@ -114,7 +115,10 @@ public void finalizeUpload(
       @PathVariable(value = "object-id") String objectId,
       @RequestParam(value = "uploadId", required = true) String uploadId
       ) {
+    val startMs = System.currentTimeMillis();
```

Comment:
Please use `Stopwatch` for this as it is cleaner and the output is much more readable for humans:

```java
val watch = Stopwatch.createStarted();
//...
log.info("Finished ... in {}", watch);
```